### PR TITLE
MAASENG-2182 address cypress intermittent failures

### PR DESCRIPTION
**Pull request by Jay-Topher**
## Done

- fix failing machine actions cypress tests

## Fixes

Fixes: [MAASEENG-2182](https://warthogs.atlassian.net/browse/MAASENG-2182)

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -54,7 +54,9 @@ Cypress.Commands.add("deleteMachine", (hostname: string) => {
   cy.visit(generateMAASURL("/machines"));
   cy.findByRole("combobox", { name: "Group by" }).select("No grouping");
   cy.findByRole("searchbox").type(hostname);
-  cy.findByText(/Showing 1 out of 1 machines/).should("exist");
+  cy.findByText(/Showing 1 out of 1 machines/, { timeout: 15000 }).should(
```
Comment (petermakowski) [inline cypress/support/commands.ts:57]:
```suggestion
  cy.findByText(/Showing 1 out of 1 machines/, { timeout: LONG_TIMEOUT }).should(
```

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -146,7 +148,9 @@ Cypress.Commands.add("waitForPageToLoad", () => {
 
 Cypress.Commands.add("waitForTableToLoad", ({ name } = { name: undefined }) => {
   cy.findByRole("grid", { name: /Loading/i }).should("exist");
-  cy.findByRole("grid", { name: /Loading/i }).should("not.exist");
+  cy.findByRole("grid", { name: /Loading/i, timeout: 15000 }).should(
```
Comment (petermakowski) [inline cypress/support/commands.ts:151]:
```suggestion
  cy.findByRole("grid", { name: /Loading/i, timeout: LONG_TIMEOUT }).should(
```

File: cypress/support/commands.ts (hunk 1 above)
Comment (petermakowski) [reply cypress/support/commands.ts:57]: We have a `LONG_TIMEOUT` constant in `cypress/constants.ts` which would be suitable here.

**Comment by petermakowski** [comment]
@Jay-Topher Could you please create a new branch based on this where you edit cypress workflow so that it runs on that branch (and not just `main`). That way we can verify the fix has worked.

**Comment by Jay-Topher** [comment]
Sure, doing that now

**Comment by Jay-Topher** [comment]
@petermakowski #5178 is a draft PR that should run the cypress tests

**Review by petermakowski — APPROVED** [review]

**Pull request merged**